Failed to load .NET runtime - Error message has wrong version

### Tested versions

- Reproducable in Godot 4.4 dev 7 [46c8f8c5c](https://github.com/godotengine/godot/commit/46c8f8c5c5874c7c56ea5b1384259de9402d3449).


### System information

Godot v4.4.dev7.mono - Windows 10 (build 19045) - Multi-window, 1 monitor - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1070 (NVIDIA; 32.0.15.6094) - Intel(R) Core(TM) i5-7640X CPU @ 4.00GHz (4 threads)

### Issue description

The error message for missing .NET install requests version 6.0, but actually needs 8.0. This appeared when attempting to run 4.4 dev 7 with .Net 7.0 installed, and confused me for a bit. Installing .NET 9.0 made it stop appearing on application start.

![Image](https://github.com/user-attachments/assets/c10fbb88-caea-45cb-97ed-5fc81c5951a1)

Also, I don't usually create GitHub issues, but I knew I could document a small issue like this. Let me know if I've done anything improperly.

### Steps to reproduce

Start godot 4.4 dev 7, without .NET 8.0+ installed.

### Minimal reproduction project (MRP)

N/A. Occurs before opening a project.

Hmm... ran into an additional C# hiccup, that maybe I should make a separate issue for?
![Image](https://github.com/user-attachments/assets/cb3c954b-361e-43d9-9a75-905f008b46b7)

- Tried NuGet regen: https://github.com/godotengine/godot/issues/81940#issuecomment-1727020579
- Tried NuGet manual fetch, got error logs below: https://github.com/godotengine/godot/issues/58955#issuecomment-2429907991

```
PS D:\daem> dotnet add package Godot.NET.Sdk
D:\daem\Daem.csproj : error : Could not resolve SDK "Godot.NET.Sdk". Exactly one of the probing messages below indicate
s why we could not resolve the SDK. Investigate and resolve that message to correctly specify the SDK.
D:\daem\Daem.csproj : error :   SDK resolver "Microsoft.DotNet.MSBuildWorkloadSdkResolver" returned null.
D:\daem\Daem.csproj : error :   Unable to find package Godot.NET.Sdk with version (= 4.4.0-dev.7)
D:\daem\Daem.csproj : error :   - Found 106 version(s) in nuget.org [ Nearest version: 4.4.0-dev.6 ]
D:\daem\Daem.csproj : error :   - Found 0 version(s) in Microsoft Visual Studio Offline Packages
D:\daem\Daem.csproj : error MSB4236: The SDK 'Godot.NET.Sdk/4.4.0-dev.7' specified could not be found.
Unable to create dependency graph file for project 'D:\daem\Daem.csproj'. Cannot add package reference.
```

 4.4.0-dev.7 doesn't seem to exist on nuget yet. Closest version is 4.4.0-dev.6

I'm willing to bet it's still uploading/propagating, seeing as dev 7 released today. Will check again later. Excited to have jolt in the engine!

--- a/modules/mono/mono_gd/gd_mono.cpp
+++ b/modules/mono/mono_gd/gd_mono.cpp
@@ -363,7 +363,7 @@ godot_plugins_initialize_fn initialize_hostfxr_and_godot_plugins(bool &r_runtime
 
 	if (load_assembly_and_get_function_pointer == nullptr) {
 		// Show a message box to the user to make the problem explicit (and explain a potential crash).
-		OS::get_singleton()->alert(TTR("Unable to load .NET runtime, no compatible version was found.\nAttempting to create/edit a project will lead to a crash.\n\nPlease install the .NET SDK 6.0 or later from https://dotnet.microsoft.com/en-us/download and restart Godot."), TTR("Failed to load .NET runtime"));
+		OS::get_singleton()->alert(TTR("Unable to load .NET runtime, no compatible version was found.\nAttempting to create/edit a project will lead to a crash.\n\nPlease install the .NET SDK 8.0 or later from https://get.dot.net and restart Godot."), TTR("Failed to load .NET runtime"));
 		ERR_FAIL_V_MSG(nullptr, ".NET: Failed to load compatible .NET runtime");
 	}
 
@@ -551,7 +551,7 @@ void GDMono::initialize() {
 #else
 
 		// Show a message box to the user to make the problem explicit (and explain a potential crash).
-		OS::get_singleton()->alert(TTR("Unable to load .NET runtime, specifically hostfxr.\nAttempting to create/edit a project will lead to a crash.\n\nPlease install the .NET SDK 6.0 or later from https://dotnet.microsoft.com/en-us/download and restart Godot."), TTR("Failed to load .NET runtime"));
+		OS::get_singleton()->alert(TTR("Unable to load .NET runtime, specifically hostfxr.\nAttempting to create/edit a project will lead to a crash.\n\nPlease install the .NET SDK 8.0 or later from https://get.dot.net and restart Godot."), TTR("Failed to load .NET runtime"));
 		ERR_FAIL_MSG(".NET: Failed to load hostfxr");
 #endif
 	}

